fintech-examples/fintech-ui/src/app/dashboard/show-transactions/show-transactions.component.ts

```diff
@@ -0,0 +1,41 @@
+import { Component, Input, OnDestroy, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { AisService } from '../services/ais.service';
+import { Subscription } from 'rxjs';
+import { AccountReference, AccountReport, TransactionsResponse } from '../../api';
+
+@Component({
+  selector: 'app-show-transactions',
+  templateUrl: './show-transactions.component.html',
+  styleUrls: ['./show-transactions.component.scss']
+})
+export class ShowTransactionsComponent implements OnInit, OnDestroy {
```

Comment:
Better naming would be TransactionListComponent according to naming conventions

Reply:
done